### PR TITLE
change snake_case to camelCase

### PR DESCRIPTION
**Pull request by vmidyllic**
add merklizedRootPosition

### CODE REVIEW

File: credentials/common-v2.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+[comment]: <> (_[proof]&# 40;# _proof&# 41;_)
+
+# proof
+
+proof contains information that is required for verification of credential.
+supported type for Iden3Credentials: _Iden3SparseMerkleTreeProof_
+
+Proof example   
+```
```
Comment (ilya-korotya) [inline credentials/common-v2.md:9]: add json word for syntaxes color

File: credentials/common-v2.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+```
+credentialSchema subject example
+"credentialSchema": {
+"@id": "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json/KYCAgeCredential.json",
+"type": "JsonSchemaValidator2018"
+}
+```
+
+# expiration
```
Comment (ilya-korotya) [inline credentials/common-v2.md:86]: expirationDate?

File: proofs/BJJSignature2021-v2.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
+<h2> <i> BJJ proof example </i> </h2>
+
+
+```json
+{
+  "@type": "BJJSignature2021",
+  "issuerData": {
+    "id": "114WWtmHRG26ea4HgZMhj6hP2P8F9fmFA5DKG1rSea",
```
Comment (ilya-korotya) [inline proofs/BJJSignature2021-v2.md:8]: we are using did now

File: proofs/BJJSignature2021-v2.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+Time in Unix when signature has been created
+
+
+# mtp
+
+Sparse merkle tree proof of the auth claim of the issuer
+
+# revocationStatus
+
+URL to fetch revocation status of the issuer auth claim
```
Comment (ilya-korotya) [inline proofs/BJJSignature2021-v2.md:77]: Can be an object. Not only URL

**Review by demonsh — APPROVED** [review]

File: credentials/common-v2.md (hunk 1 above)
Comment (vmidyllic) [reply credentials/common-v2.md:9]: fixed

File: schemas/json-ld/kyc-v3.json-ld [hunk 5]
```diff
@@ -0,0 +1,48 @@
+{
+  "@context": [
+    {
+      "@version": 1.1,
+      "@protected": true,
+      "id": "@id",
+      "type": "@type",
+      "KYCAgeCredential": {
+        "@id": "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/kyc.json-ld#KYCAgeCredential",
```
Comment (ilya-korotya) [inline schemas/json-ld/kyc-v3.json-ld:9]: kyc-v3.json-ld

File: schemas/json-ld/kyc-v3.json-ld [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+            "@type": "xsd:integer"
+          },
+          "documentType": {
+            "@id": "kyc-vocab:documentType",
+            "@type": "xsd:integer"
+          }
+        }
+      },
+      "KYCCountryOfResidenceCredential": {
+        "@id": "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/kyc.json-ld#KYCCountryOfResidenceCredential",
```
Comment (ilya-korotya) [inline schemas/json-ld/kyc-v3.json-ld:28]: kyc-v3.json-ld

File: schemas/json/KYCCountryOfResidenceCredential-v1.json [hunk 7]
```diff
@@ -0,0 +1,133 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "$metadata": {
+    "uris": {
+      "jsonLdContext": "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/kyc-v3.json-ld",
+      "jsonSchema": "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json/KYCCountryOfResidenceCredential-v2.json"
```
Comment (ilya-korotya) [inline schemas/json/KYCCountryOfResidenceCredential-v1.json:7]: File name `KYC...Credential-v1.json` but here `KYC...Credential-v2.json`

File: schemas/json/KYCCountryOfResidenceCredential-v2.json [hunk 8]
```diff
@@ -0,0 +1,133 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "$metadata": {
+    "uris": {
+      "jsonLdContext": "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/kyc-v3.json-ld",
+      "jsonSchema": "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json/KYCCountryOfResidenceCredential-v3.json"
```
Comment (ilya-korotya) [inline schemas/json/KYCCountryOfResidenceCredential-v2.json:7]: file v2 - in `jsonSchema` v3

File: schemas/json/auth-v1.json [hunk 9]
```diff
@@ -0,0 +1,137 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$metadata": {
+    "uris": {
+      "jsonLdContext": "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/auth-v2.json-ld",
```
Comment (ilya-korotya) [inline schemas/json/auth-v1.json:5]: `jsonLdContext` and `jsonSchema` has different versions

File: schemas/json/KYCCountryOfResidenceCredential-v2.json (hunk 8 above)
Comment (vmidyllic) [reply schemas/json/KYCCountryOfResidenceCredential-v2.json:7]: fixed

File: schemas/json/KYCCountryOfResidenceCredential-v1.json (hunk 7 above)
Comment (vmidyllic) [reply schemas/json/KYCCountryOfResidenceCredential-v1.json:7]: fixed

File: schemas/json/auth-v1.json (hunk 9 above)
Comment (vmidyllic) [reply schemas/json/auth-v1.json:5]: fixed

Comment (vmidyllic) [reply schemas/json/auth-v1.json:5]: that's ok

File: schemas/json-ld/kyc-v3.json-ld (hunk 5 above)
Comment (vmidyllic) [reply schemas/json-ld/kyc-v3.json-ld:9]: fixed

**Review by ilya-korotya — APPROVED** [review]

**Pull request merged**